rules/linux/command_and_control_linux_kworker_netcon.toml

```diff
@@ -113,7 +113,7 @@ framework = "MITRE ATT&CK"
 
 [rule.new_terms]
 field = "new_terms_fields"
-value = ["host.id", "process.name", "destination.ip"]
+value = ["process.name", "destination.ip", "destination.port"]
```

Comment:
Do we want to remove the per host condition?

Reply:
Yep, in this case, I do, as most of the noise was related to similar kernel thread processes running on different hosts.